src/lib/auth.ts

```
@@ -62,9 +62,10 @@ export async function loginUser(data: LoginData) {
     }
 
     throw new Error("Credenciales inválidas");
-  } catch (error) {
+  } catch (error: any) {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)